diff --git a/node_modules/fstream/lib/collect.js b/node_modules/fstream/lib/collect.js
index e5d4f35..3248314 100644
--- a/node_modules/fstream/lib/collect.js
+++ b/node_modules/fstream/lib/collect.js
@@ -12,7 +12,7 @@ function collect (stream) {
   stream.on('end', save)
   var buf = []
   function save (b) {
-    if (typeof b === 'string') b = new Buffer(b)
+    if (typeof b === 'string') b = Buffer.alloc(b)
     if (Buffer.isBuffer(b) && !b.length) return
     buf.push(b)
   }
