diff --git a/node_modules/ip/lib/ip.js b/node_modules/ip/lib/ip.js
index c1799a8..a66a255 100644
--- a/node_modules/ip/lib/ip.js
+++ b/node_modules/ip/lib/ip.js
@@ -10,7 +10,7 @@ ip.toBuffer = function(ip, buff, offset) {
   var result;
 
   if (this.isV4Format(ip)) {
-    result = buff || new Buffer(offset + 4);
+    result = buff || Buffer.alloc(offset + 4);
     ip.split(/\./g).map(function(byte) {
       result[offset++] = parseInt(byte, 10) & 0xff;
     });
@@ -45,7 +45,7 @@ ip.toBuffer = function(ip, buff, offset) {
       sections.splice.apply(sections, argv);
     }
 
-    result = buff || new Buffer(offset + 16);
+    result = buff || Buffer.alloc(offset + 16);
     for (i = 0; i < sections.length; i++) {
       var word = parseInt(sections[i], 16);
       result[offset++] = (word >> 8) & 0xff;
@@ -110,7 +110,7 @@ ip.fromPrefixLen = function(prefixlen, family) {
   if (family === 'ipv6') {
     len = 16;
   }
-  var buff = new Buffer(len);
+  var buff = Buffer.alloc(len);
 
   for (var i = 0, n = buff.length; i < n; ++i) {
     var bits = 8;
@@ -129,7 +129,7 @@ ip.mask = function(addr, mask) {
   addr = ip.toBuffer(addr);
   mask = ip.toBuffer(mask);
 
-  var result = new Buffer(Math.max(addr.length, mask.length));
+  var result = Buffer.alloc(Math.max(addr.length, mask.length));
 
   var i = 0;
   // Same protocol - do bitwise and
