diff --git a/node_modules/json-buffer/index.js b/node_modules/json-buffer/index.js
index 9cafed8..c8cc73c 100644
--- a/node_modules/json-buffer/index.js
+++ b/node_modules/json-buffer/index.js
@@ -49,7 +49,7 @@ exports.parse = function (s) {
   return JSON.parse(s, function (key, value) {
     if('string' === typeof value) {
       if(/^:base64:/.test(value))
-        return new Buffer(value.substring(8), 'base64')
+        return Buffer.alloc(value.substring(8), 'base64')
       else
         return /^:/.test(value) ? value.substring(1) : value 
     }
